java/fury-core/src/main/java/io/fury/config/Config.java

```diff
@@ -150,6 +152,10 @@ public boolean requireClassRegistration() {
     return requireClassRegistration;
   }
 
+  public boolean classRegistrationWarningIgnored() {
```

Comment:
```suggestion
  public boolean isClassRegistrationWarningIgnored() {
```

Also better to add java doc for this method here